demisto_sdk/commands/download/downloader.py

```diff
@@ -1393,6 +1396,24 @@ def download_new_content_items(
                 if file_path.is_file():
                     downloaded_files.append(file_path)
 
+        elif content_item_entity_directory == LISTS_DIR:
```

Comment:
what is this change in the download command? do we need to document it too in the changelog?

Reply:
This is definitely a change in **_download_** for `content-type list`
from now on during the download of a list it will be split

I will add it as a breaking change to the `CHANGELOG`